src/pages/app/posting_project_user/ProfilePage.js

```diff
@@ -121,40 +151,61 @@ const ProfilePage = (props) => {
     projectId: null,
     projectName: null,
   };
+  let profileData = {
+    orgId: null,
+    orgName: null,
+    location: null,
+    university: null,
+    category: null,
+    description: null,
+    email: null,
+    facebook: null,
+    phoneNumber: null,
+    vision: null,
+  };
 
   //Declare useDispatch hook to use;
   const dispatch = useDispatch();
-  //Load profileData from global store into 'selectedData' variable using useSelector() hook
-  const selectedData = useSelector(
+
+  //Load data from global store into variables variable using useSelector() hook
+  const selectedProject = useSelector(
+    (state) => state.profileReducer.projectData.props
+  );
+  const selectedProfile = useSelector(
     (state) => state.profileReducer.profileData.props
   );
+
   //Update the data from the 'selectedData' variable into the local variable
-  //If the data is not equal to undefined (this mean the data is fetched succesfully) then
-  //assign it into the local variable.
-  //else, execute the FetchData() function.
-  if (selectedData != undefined) {
-    profileData = selectedData;
-  } else FetchData();
-  console.log("This is ProfileData name", profileData.orgName); // For testing purpose
-  console.log("Profile data saved succesfully:", profileData); //For testing purpose
-  //Function to fetch data and dispatch it into the global store. I will move this is the profileAction.js later
+  //If the data is equal to undefined (not fetched succesfully) then execute the FetchData() function.
+  if (selectedProject != undefined) {
+    projectData = selectedProject;
+  } else FetchData("public-projects", `${testing_project_id}`);
+  console.log("Profile data saved succesfully:", profileData.orgName); //For testing purpose
+  //Similar for Profile Data
+  if (selectedProfile != undefined) {
+    profileData = selectedProfile;
+  } else FetchData("organization", `${testing_organization_id}`);
+  console.log("Profile data saved succesfully:", profileData.orgName); //For testing purpose
+
+  //Function to fetch data and dispatch it into the global store.
   //The function need to use async and await to be able to fetch.
-  async function FetchData() {
+  async function FetchData(collection, fetchId) {
     let fetchedData; //This will later be used to fetch data.
     console.log("Starting to fetch data: "); //For testing purpose
-    //try and catch (from the concept promises) is to catch error.
+    console.log("Params: ", collection, " + ", fetchId); //For testing purpose
```

Comment:
If the codes have the different roles, so you should have 2 or 4 backspaces

Reply:
Well in my eyes this looks good. I mean, it looks terrible surely, but still less terrible than if I were to put 2 or 4 enter line.

Reply:
Put 2 or 4 , suppose in line 189 maybe easier to read